scripts/build/check_init_priorities.py

```diff
@@ -54,6 +54,14 @@
 # opposite of the device tree inferred dependency.
 _INVERTED_PRIORITY_COMPATIBLES = frozenset()
 
+# List of compatibles for nodes where we don't check the priority.
+_IGNORE_COMPATIBLES = frozenset([
+        # This is pre-initialized early to start processing data as soon as
+        # possible, there's a separate SYS_INIT entry running later in the boot
+        # sequence.
```

Comment:
@jfischer-no I took a good look at the code and I think it makes sense to just not check this, at least for now, does this sounds like a right summary of how https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/usb/device_next/class/usbd_cdc_acm.c#L1128 works?

Reply:
This is not quite right, and we will probably change or remove SYS_INIT(). One thing is when the virtual driver is initialized, another is when the connection to the host is established, which is always later or never. I would write "There is no direct dependency between the CDC ACM UART and the USB device controller, the logical connection is established after USB device support is enabled. "

Reply:
Cool, I've reworded it with your description, thanks!